### PR TITLE
List container component

### PR DESCRIPTION
**Pull request by kimmichurri**
Return the matched climbs based on type and area to be displayed now by description component. Ready to work on description component now.

### CODE REVIEW

**Comment by Inzurriaga** [comment]
@robbiejaeger @brittanystoroz  @thatPamIAm idk if you can still see it but this is are component for the list container with the basic concept of how it should work

**Comment by codexguajil** [comment]
Good work! Kim, you and I will have to make List Container a class component to implement my Search string results!

**Comment by timmiller601** [comment]
Great job - I will be using matchType prop to set up the description

File: src/Filter.js [hunk 1]
```diff
@@ -63,6 +63,7 @@ export default class Filter extends Component {
 
   submitFilter = event => {
     event.preventDefault()
+    console.log('submit button')
```
Comment (robbiejaeger) [inline src/Filter.js:66]: Be sure to remove this `console.log` before merging.

File: src/ListContainer.js [hunk 2]
```diff
@@ -3,16 +3,48 @@ import Description from './Description';
...
+  // })
+
+  let matchedAreas = Object.keys(props.areasAndRoutes).reduce((acc, area) => {
+    if (props.areaName.includes(area)) {
+     acc.push(props.areasAndRoutes[area])
+    }
+    return acc;
+  }, [])
+
+  console.log(matchedAreas, "areas");
```
Comment (robbiejaeger) [inline src/ListContainer.js:17]: Same here with the `console.log()`s.

File: src/ListContainer.js [hunk 3]
```diff
@@ -3,16 +3,48 @@ import Description from './Description';
...
-            return (
-              <Description distance={distance}/>
-              
-            )
-          })
-        }
+          
+      matchedType.map((type) => {
+        return (
+          <Description matchedType={type}
```
Comment (robbiejaeger) [inline src/ListContainer.js:42]: Are you getting any warnings in the dev tools console about needing a unique key? If so, check out this doc page to see why and what to do: https://reactjs.org/docs/lists-and-keys.html#keys

**Comment by robbiejaeger** [comment]
I added a couple comments to this PR. Be sure to address them in your next PR since this has already been merged.

**Pull request merged**